docs/v0.5/getting-started/docker-for-windows.md

```diff
@@ -211,22 +189,26 @@ riff doctor
 ```
 
 ```
-NAMESPACE     STATUS
+default       ok
```

Comment:
```suggestion
NAMESPACE     STATUS
default       ok
```
Did you mean to delete the headers here?

Reply:
oops, no I didn't